src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let daysSpend = days * 40;
+
+  if (days >= 3 && days < 7) {
+    daysSpend -= 20;
+
+    return daysSpend;
+  } else if (days >= 7) {
+    daysSpend -= 50;
+
+    return daysSpend;
+  } else {
+    return daysSpend;
```

Comment:
This `return` statement is also repetitive. Consider having a single `return daysSpend;` at the end of the function.